runtime/plugin/rplugin.vim

```diff
@@ -5,12 +5,14 @@ let g:loaded_remote_plugins = 1
 
 command! UpdateRemotePlugins call remote#host#UpdateRemotePlugins()
 
-augroup nvim-rplugin
-  autocmd!
-  autocmd FuncUndefined *
-        \ call remote#host#LoadRemotePluginsEvent(
-        \   'FuncUndefined', expand('<amatch>'))
-  autocmd CmdUndefined *
-        \ call remote#host#LoadRemotePluginsEvent(
-        \   'CmdUndefined', expand('<amatch>'))
-augroup END
+call remote#host#LoadRemotePlugins()
```

Comment:
This autoloads the whole autoload/remote stuff. Instead I moved `s:LoadRemotePlugins()` to this file.